### PR TITLE
ENG-2063 plugin descriptor version >= 3 generates an ingressPath with… (also ENG-2281)

### PR DESCRIPTION
**Pull request by firegloves**
…out the docker image version

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **firegloves**

If they are not already assigned, you can assign the PR to them by writing `/assign @firegloves` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-component-manager).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-component-manager/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-component-manager-ecmpr123**

File: src/main/java/org/entando/kubernetes/model/bundle/processor/PluginProcessor.java
Comment (w-caffiero-entando) [inline src/main/java/org/entando/kubernetes/model/bundle/processor/PluginProcessor.java:157]: @firegloves constantize the "v" and/or create a function for this composition

File: src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java
Comment (w-caffiero-entando) [inline src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java:254]: @firegloves hande NumberFormatException and rethrow as InvalidBundleException with a proper message

File: src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java
Comment (w-caffiero-entando) [inline src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java:273]: @firegloves public functions always need javadoc

File: src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java
Comment (w-caffiero-entando) [inline src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java:267]: @firegloves public functions always need javadoc

Comment (w-caffiero-entando) [reply src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java:267]: see also the rest of the public functions contained in this PR

File: src/main/java/org/entando/kubernetes/model/bundle/processor/PluginProcessor.java
Comment (w-caffiero-entando) [inline src/main/java/org/entando/kubernetes/model/bundle/processor/PluginProcessor.java:201]: @firegloves 
My proposal:
> The plugin "{{INSERT PLUGIN NAME}}" descriptor contains an invalid descriptorVersion

File: src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java
Comment (firegloves) [reply src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java:254]: @w-caffiero-entando I'm adding it but the check is already made with the regex, don't it?

Comment (w-caffiero-entando) [reply src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java:254]: yeah, right, missed it

**Comment by entando-jx** [comment]
@firegloves: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | ae7a7147db6a5a46be11ebaa755302d26c059d86 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
@firegloves: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 93601d0470c33ed9f8c242cac2a1c466091ff3ef | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-component-manager-ecmpr123**

File: src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java
Comment (joewhite101) [inline src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java:270]: There should be logging to go with this. Even if the exception is logged elsewhere.

File: src/main/java/org/entando/kubernetes/model/bundle/processor/PluginProcessor.java
Comment (joewhite101) [inline src/main/java/org/entando/kubernetes/model/bundle/processor/PluginProcessor.java:168]: There should be logging to go with this. Even if the exception is logged elsewhere.

**Review by w-caffiero-entando — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-component-manager/pull/123#pullrequestreview-644683575" title="Approved">w-caffiero-entando</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-component-manager).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando-k8s/entando-component-manager/blob/master/OWNERS)~~ [w-caffiero-entando]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
